front/tests/pages/rollingstock-modal-model.ts

```diff
@@ -41,12 +41,9 @@ class PlaywrightRollingstockModalPage {
     await this.getRollingStockSelector.click();
   }
 
-  async checkNumberOfRollingstockFound(expectedRollingstock: string) {
-    await expect(this.getResultsFound).toHaveText(
-      `${expectedRollingstock} ${this.playwrightCommonPage.getRollingstockTranslation(
-        'resultsFound'
-      )}`
-    );
+  async checkNumberOfRollingstockFound() {
```

Comment:
Here you check if there is at least one rollingstock but not a fixed value, so you should rename the function `isAnyRollingstockFound` ?